code/perception/launch/perception.launch

```diff
@@ -26,4 +26,14 @@
     <param name="debug" value="true"/>
     <param name="self_diagnose" value="true"/>
   </node>
+
+  <node pkg="perception" type="lidar_distance.py" name="lidar_distance" output="screen">
+    <param name="max_y" value="1.5"/>
```

Comment:
what about max_x?

Reply:
You only have to specify parameters you need. max_x would be the limit in front of the car. I think it is not necessary to limit this direction since we want to get the closest point in front of us, even if it's far away? :)

Reply:
Yes i agree. I had a wrong understanding of the parameters. The picture helped me a lot